.babelrc

```diff
@@ -1,3 +1,10 @@
 {
-  "presets": ["es2015"]
+  "presets": [
+    ["env", {
+      "targets": {
+        "node": "4"
+      },
+      "exclude": ["transform-regenerator"]
+    }]
+  ]
 }
```

Comment:
this is a good change; can you make a separate PR to the main repo switching to the env preset?

Reply:
ok. Filled https://github.com/sarbbottam/eslint-find-rules/pull/340

Reply:
will start rebase after https://github.com/sarbbottam/eslint-find-rules/pull/337. 👀